scripts/check-format.sh

```diff
@@ -32,9 +32,9 @@ file_name_regex="^[[:lower:]0-9_]+$"
 unformatted_files=""
 badly_named_files=""
 for file in $(git ls-files "$@" | grep -e '\.h$' -e '\.hpp$' -e '\.cpp$' -e '\.c$' -e '\.proto$'); do
-  if ! clang-format-10 -n -Werror -style=file "$file"; then
+  if ! clang-format-11 -n -Werror -style=file "$file"; then
```

Comment:
Since you have upgraded clang-format too, I think you would need to format all the files with clang-format-11, since some formatting rules might have changed from clang-format-10.